usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -322,36 +881,6 @@
    LEFT OUTER JOIN
         (SELECT id, toptier_agency_id, ROW_NUMBER() OVER (PARTITION BY toptier_agency_id ORDER BY toptier_flag DESC, id ASC) AS row_num FROM global_temp.agency) AS FA_ID
         ON (FA_ID.toptier_agency_id = TFA.toptier_agency_id AND row_num = 1)
-    LEFT OUTER JOIN
-        global_temp.naics ON (transaction_fpds.naics = naics.code)
-    LEFT OUTER JOIN
-        global_temp.psc ON (transaction_fpds.product_or_service_code = psc.code)
-    LEFT OUTER JOIN (
```

Comment:
For the location reference data we might want to keep a "raw" and "current" version. The location data is not historically the best in its raw form. For example, if you look at the values for `legal_entity_country_name` many of those are blank but the `legal_entity_country_code` is populated.

Reply:
Going to resolve this conversation since search functionality is on the location codes instead of the names which should not be an issue. Something to be aware of as we create the different views on this table that might be displaying the name values on the site.